ui/src/components/setting/SettingBilling.vue

```diff
@@ -217,6 +253,18 @@ export default {
       return this.$store.getters['namespaces/owner'];
     },
 
+    processed() {
+      return this.$store.getters['billing/status'] === 'processed';
+    },
+
+    inactive() {
```

Comment:
I don't see the need to create another computed, as there is only one that checks if it is active.

Reply:
Several places are checking the same store variable, I could redo that logic. This just adds more load to the UI.